files-community/Files
# Fix: Fixed support for High Contrast themes

**Resolved / Related Issues**
- Fixed high contrast support

**Validation**
How did you test these changes?
   1. Enable one of the high contrast themes in Windows
   2. Confirm the icons display properly

**Screenshots (optional)**
Add screenshots here.
